test/migrators/division/always_division.hrx

```diff
@@ -0,0 +1,37 @@
+<==> input/entrypoint.scss
+@function six-divided-by-three() {
+  @return 6px / 3px;
+}
+
+a {
+  b: (4px + 2px) / 3px;
+  c: 6px/3px + 1;
+  d: (6px / 3px);
+  e: 6px / (2px + 1px) / 2;
+  $f: 6px / 3px;
+
+  // * and % always return numbers, so we can assume division here
```

Comment:
I think it's worth testing all the operators that force numeric mode in all the positions they can appear.